### PR TITLE
Cleanup debugging

### PR DESCRIPTION
**Pull request by ineiti**
Cleans up some of the now unused debugging things and 'dev' Dockerfiles.

### CODE REVIEW

File: .github/workflows/go_integration_tests.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
     branches: [main]
 
 jobs:
+  if: false
```
Comment (PascalinDe) [inline .github/workflows/go_integration_tests.yml:10]: please see if expected failures are a thing and replace it by that if possible

File: docker-compose/docker-compose.dev.yml
Comment (PascalinDe) [inline docker-compose/docker-compose.dev.yml:1]: please remove file from PR

File: docker-compose/docker-compose.yml
Comment (PascalinDe) [inline docker-compose/docker-compose.yml:1]: please remove this file from PR

File: scripts/init_dela.sh [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
+  ln -s ../.env ../docker-compose/.env
+fi
 
-
-# share the certificate
+echo "Share the certificate"
 for container in dela-worker-1 dela-worker-2 dela-worker-3; do
   TOKEN_ARGS=$(docker compose exec dela-worker-0 /bin/bash -c 'LLVL=error dvoting --config /data/node minogrpc token');
-  docker compose exec "$container" dvoting --config /data/node minogrpc join --address //dela-worker-0:2000 $TOKEN_ARGS;
+  docker compose exec "$container" dvoting --config /data/node minogrpc join --address //localhost:2000 $TOKEN_ARGS;
```
Comment (PascalinDe) [inline scripts/init_dela.sh:12]: please discard the change in this line

File: scripts/init_dela.sh
Comment (PascalinDe) [inline scripts/init_dela.sh:1]: please discard the changes

File: scripts/setup_forms.sh
Comment (PascalinDe) [inline scripts/setup_forms.sh:1]: please do not include this file

File: scripts/setup_proxies.sh
Comment (PascalinDe) [inline scripts/setup_proxies.sh:1]: please discard these changes

**Review by lanterno — APPROVED** [review]

**Comment by lanterno** [comment]
@ineiti You need to rebase

**Comment by ineiti** [comment]
@lanterno done.

**Review by pierluca — COMMENTED** [review]
Why remove the debug dockerfiles ? It was super painful to get it just right 😅

**Comment by ineiti** [comment]
> Why remove the debug dockerfiles ? It was super painful to get it just right 😅

We never actually used them, and I'm preparing #366 which allows running a whole system locally w/o docker, similar to `runSystems.sh`, but ready for voting. So you can also directly run a node with Goland without having to go through docker.

But if you promise you'll use it, I'll put it back in.

**Comment by pierluca** [comment]
Err, I did actually use it with @PascalinDe . I think it doesn't hurt to keep it around.
I appreciate the #366 initiative (very much so, it looks very neat!), but I'm a little confused - I was under the impression that @PascalinDe 's effort to containerize the environment was that we could (eventually) get rid of those scripts and develop with the convenience of a pre-production environment ?

**Comment by ineiti** [comment]
Different environments are needed for different types of development. @PascalinDe 's efforts to containerize the environment supposed we only need to do small debugging and adjustments. As it turned out, there were a lot of things to fix and change, and for faster debugging cycles I prefer having everything in the shell w/o docker.

Once it works locally, it's nice to have a docker environment to test it on other machines, too, without them having to install all the development environment.

So it's two approaches for two different development use-cases.

Adding the debug Dockerfile in again.

**Comment by PascalinDe** [comment]
@pierluca we are currently working on a Dockerfile that includes the debug environment: https://github.com/dedis/d-voting/pull/364/files

**Comment by ineiti** [comment]
Closing in favour of c4dt/d-voting#3

**Pull request closed without merging**